project_template/config/karma.js

```diff
@@ -1,8 +1,4 @@
 const webpackConfig = require("../webpack.config.js");
-const commonsChunkPluginIndex = webpackConfig.plugins.findIndex(
```

Comment:
@ruisalgado do you remember why this was needed earlier?

Reply:
@pascalw nope. Seems like we're trying to exclude some plugins from being loaded by karma but... no idea why